pkg/reconciler/taskrun/taskrun.go

```diff
@@ -349,7 +352,7 @@ func (c *Reconciler) prepare(ctx context.Context, tr *v1.TaskRun) (*v1.TaskSpec,
 		if resources.IsErrTransient(err) {
 			return nil, nil, err
 		}
-		tr.Status.MarkResourceFailed(podconvert.ReasonFailedResolution, err)
+		tr.Status.MarkResourceFailed(v1.TaskRunReasonFailedResolution, err)
```

Comment:
Should we avoid using `podconvert.xxx` error reason when it is just a alias to `v1.xxx` error reason (for example line 346). My understanding the alisa is just for backward compatability and should no longer be used in our codebase

Reply:
Yes that is correct. Those reasons were once living in `pkg/reconciler` and moved to `pkg/pod`. But they are actually tied with api since it is part of [the `apis.condition` reason for TaskRunStatus](https://github.com/tektoncd/pipeline/blob/f88a6433f1863a8e005eba3fbbcd54e4c3d35c2c/pkg/apis/pipeline/v1/taskrun_types.go#L229). I tried to call out this at [#L41](// Aliased for backwards compatibility; do not add additional TaskRun reasons here).

Reply:
I've also updated all reference to podconvert.*reasons to use v1.*TaskRunReasons instead.